src/instance.ts

```diff
@@ -56,20 +58,15 @@ import {ServiceError} from 'google-gax';
 import {Bigtable} from '.';
 import {google} from '../protos/protos';
 
-export interface ClusterInfo {
-  id?: string;
-  location?: string;
-  serveNodes?: number;
-  nodes?: number;
-  storage?: string;
-  defaultStorageType?: number;
```

Comment:
BREAKING CHANGE
Removing `serveNodes` and `defaultStorageType` as these are expected by GAPIC client, user facing parameters are `nodes` and `storage` respectively.